<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-S15VDGS" failures="1" tests="4" name="Test" time="65.175" errors="0" timestamp="2024-05-29T11:20:41 GMT-07:00">
  <testcase classname="testcases.RegisterTestCases" name="VerifyRegistrationWithValidData" time="3.991"/>
  <testcase classname="testcases.RegisterTestCases" name="VerifyRegistrationWithBlankData" time="3.009"/>
  <testcase classname="testcases.LoginTestCases" name="VerifyLoginWithInvlidData" time="3.245">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Warning: No match for E-Mail Address and/or Password.] but found [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Warning: No match for E-Mail Address and/or Password.] but found [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.]
at resources.commonMethods.handleAssertion(commonMethods.java:17)
at testcases.LoginTestCases.VerifyLoginWithInvlidData(LoginTestCases.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginWithInvlidData -->
  <testcase classname="testcases.LoginTestCases" name="VerifyLoginWithValidData" time="2.987"/>
  <testcase classname="resources.BaseClass" name="@AfterSuite endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endReport -->
  <testcase classname="resources.BaseClass" name="@BeforeSuite ExtentReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite ExtentReport -->
  <testcase classname="testcases.RegisterTestCases" name="VerifyRegistrationWithInvalidData" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyRegistrationWithInvalidData -->
</testsuite> <!-- Test -->
